src/js/edu-benefits/education-wizard.js

```diff
@@ -41,12 +41,14 @@ function reInitWidget() {
 
   content.addEventListener('change', (e) => {
     const radio = e.target;
-    const otherChoice = radio.dataset.alternate;
-    const otherNextQuestion = container.querySelector(`#${otherChoice}`).dataset.nextQuestion;
-    if (otherNextQuestion) {
-      const otherNextQuestionElement = container.querySelector(`[data-question=${otherNextQuestion}]`);
-      closeStateAndCheckChild(otherNextQuestionElement, container);
-    }
+    const otherChoices = radio.dataset.alternate.split(' ');
```

Comment:
I was misunderstanding how `data-alternate` was being used; this change makes sense.